.github/workflows/test-e2e.yml

```diff
@@ -15,50 +15,54 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                storage: [ LOCAL, MEMORY, PLATFORM ]
+                storage: [LOCAL, MEMORY, PLATFORM]
 
         steps:
-            -   name: Cancel Workflow Action
-                uses: styfle/cancel-workflow-action@0.12.0
-                with:
-                    access_token: ${{ github.token }}
+            - name: Cancel Workflow Action
+              uses: styfle/cancel-workflow-action@0.12.0
+              with:
+                  access_token: ${{ github.token }}
 
-            -   name: Checkout repository
-                uses: actions/checkout@v4
+            - name: Checkout repository
+              uses: actions/checkout@v4
 
-            -   name: Use Node.js 18
-                uses: actions/setup-node@v4
-                with:
-                    node-version: 18
-                    cache: 'yarn'
-                    cache-dependency-path: 'yarn.lock'
+            - name: Use Node.js 20
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 20
 
-            -   name: Enable corepack
-                run: |
-                    corepack enable
-                    corepack prepare yarn@stable --activate
+            - name: Enable corepack
+              run: |
+                  corepack enable
+                  corepack prepare yarn@stable --activate
 
-            -   uses: microsoft/playwright-github-action@v1
+            - name: Activate cache for Node.js 20
+              uses: actions/setup-node@v4
+              with:
+                  cache: 'yarn'
+                  cache-dependency-path: 'yarn.lock'
```

Comment:
dont we need this in all the other places too? or maybe its the default anyway and we dont need it here?

Reply:
I mean the default points to yarn.lock, not sure why we even have it here